docs/deploy_narrative.md

```diff
@@ -1,64 +1,109 @@
+    -   Add a new heading with the updated version.
+    -   Add a list of changes with some reasonable detail. You don't need to add the name of every module changed, just the overall feature affected. Include JIRA ticket numbers where appropriate.
 
-### Appdev, Next, & Prod
+3.  PR these changes to the develop branch.
+    -   Ensure that tests pass.
 
-#### Create Image
+4.  PR the develop branch to the master branch.
+    -   Deploy on the next environment.
+    -   Ensure that things work as expected.
 
-1. Create a new [pull request](https://github.com/kbase/narrative/compare) to merge the `develop` branch into `master`.
-1. Follow the same process outlined in the `Develop` section above.
+5.  Create a new release in the narrative repo.
+    -   tag it with the new version, prefixed with v (e.g. v4.2.1).
+
+### Deploying Next, Appdev, & Prod
+
+These environments: <https://next.kbase.us>, <https://appdev.kbase.us>, and <https://narrative.kbase.us> are all deployed using a production image. They're used for slightly different purposes:
+
+-   **next** is used for final internal testing before being promoted to the public-facing environments - appdev and production. Most services should be up to date here, and this is used for final integration and acceptance testing.
+-   **appdev** is used primarily for KBase app development using the [KBase SDK](https://kbase.github.io/kb_sdk_docs/). This environment mirrors the production environment, but allows for finding and running apps in "dev" mode.
+-   **prod** (or narrative.kbase.us) is the main production environment. This is the place where most users will use KBase and run apps that have either been entirely released or have been made available for beta testing.
+
+#### Create Release Image
+
+1.  Create a new [pull request](https://github.com/kbase/narrative/compare) to merge the `develop` branch into `master`.
+2.  Follow the same process outlined in the `Develop` section above.
 
 Once a pull request is merged from `develop` to `master` a new _production_ image will be available at:
 `docker.pkg.github.com/kbase/narrative/narrative:latest`.
 
-
 #### Tag & Deploy Image
 
-1. Ensure the [deploy_tag.sh](https://github.com/kbase/narrative-traefiker/blob/develop/.github/workflows/scripts/deploy_tag.sh) script is installed on either your local workstation, or under your home directory on a secured system, such as `install.kbase.lbl.gov`.
-2. Create a GitHub [personal access token](https://github.com/settings/tokens) and set it as "$TOKEN" in your workstation/server shell environment.
-3. Run the `deploy_tag.sh` with the following syntax:
-`./deploy_tag.sh -o "kbase" -r "narrative" -t "latest" -e "appdev"`
+1.  Ensure the [deploy_tag.sh](https://github.com/kbase/narrative-traefiker/blob/develop/.github/workflows/scripts/deploy_tag.sh) script is installed on either your local workstation, or under your home directory on a secured system, such as `install.kbase.lbl.gov`.
```

Comment:
perhaps add a comment under this header that this applies to KBase sys admins / devops only.

Reply:
Also a good idea. Done.